core/domain/config_domain.py

```diff
@@ -208,6 +208,7 @@ def init_config_property(cls, name, instance):
 
         Args:
             name: str. The name of the configuration property.
+            instance: Registry. The instance of the configuration property.
```

Comment:
Pretty sure the type is wrong. We don't have any objects of type Registry.

Reply:
Please see [this](https://github.com/oppia/oppia/pull/4967#discussion_r190451332) regarding the type of the instance. Thanks!

Reply:
@kevinlee12, why is Registry the type of the instance?

@apb7, to find the type, do something like `raise Exception(type(varname))` which should show the type. I think you mentioned that the types are different depending on the property, which makes sense. In that case the type should be "*".

Reply:
Done.